modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -101,49 +115,49 @@ def number_and_street
     end
 
     def city
-      veteran.dig('address', 'city') || ''
+      veteran_data.dig('address', 'city') || ''
```

Comment:
We've hooked up the veteran appellant object - let's use it for as many of these `veteran_data` calls as we can. It's exactly why Appellant was written, yeah? :)

Reply:
If you go to the "all the things" draft pr, you can see how v2 data and structure work with the new setup.